### PR TITLE
Converts to srcSet selection instead of manual aliasing

### PR DESCRIPTION
**Pull request by dzucconi**
Given that https://github.com/artsy/metaphysics/pull/2771 is deployed to production — I wanted to convert all this manual aliasing. Going forward whenever u need a resized `src`, just ask for and pass the `srcSet` as well. That's it.

### CODE REVIEW

File: src/v2/Apps/Fair/Components/FairEditorial/FairEditorialItem.tsx [hunk 1]
```diff
@@ -76,8 +76,8 @@ export const FairEditorialItem: React.FC<FairEditorialItemProps> = ({
       >
         {article.thumbnailImage && (
           <Thumbnail
-            src={article.thumbnailImage._1x.src}
-            srcSet={`${article.thumbnailImage._1x.src} 1x, ${article.thumbnailImage._2x.src} 2x`}
+            src={article.thumbnailImage.cropped.src}
+            srcSet={article.thumbnailImage.cropped.srcSet}
```
Comment (mzikherman) [inline src/v2/Apps/Fair/Components/FairEditorial/FairEditorialItem.tsx:80]: ok, first line of this PR, and i'm liking it already!

File: src/v2/Apps/Fair/Components/FairEditorial/FairEditorialItem.tsx [hunk 2]
```diff
@@ -99,15 +99,11 @@ export const FairEditorialItemFragmentContainer = createFragmentContainer(
...
             width
             height
-            src: url
-          }
-          _2x: cropped(width: 280, height: 160) {
-            width
-            height
-            src: url
+            src
+            srcSet
```
Comment (mzikherman) [inline src/v2/Apps/Fair/Components/FairEditorial/FairEditorialItem.tsx:106]: nice

**Review by anandaroop — APPROVED** [review]
👌 nice

File: src/v2/Apps/Feature/Components/FeatureFeaturedLink.tsx [hunk 3]
```diff
@@ -142,31 +142,31 @@ export const FeatureFeaturedLinkFragmentContainer = createFragmentContainer(
...
+          large: cropped(width: 1112, height: 626, version: ["main", "wide"]) {
+            src
+            srcSet
             width
             height
           }
-          # ?:? - 1112×1112 native max dimensions * 2 for retina
-          full: resized(width: 2224, height: 2224, version: ["main", "wide"]) {
-            src: url
+          # ?:?
```
Comment (mzikherman) [inline src/v2/Apps/Feature/Components/FeatureFeaturedLink.tsx:166]: lol

is this `1:1` ?

**Review by mzikherman — APPROVED** [review]
Wow, so nice!

Random thought- should we/you publicize this to the wider team so more people can use the `srcsSet` when displaying images (this might be worthy of a custom `ts-lint` rule too?)

**Pull request merged**